package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest --coverage",
     "build": "webpack -d --mode development --watch --config webpack.config.js",
-    "start": "nodemon ./server/index.js",
+    "start": "nodemon ./server/index.jsx",
```

Comment:
for the server file, you may not need to have a .jsx file format; .js is probably fine.

Reply:
Oh nice catch! I will try to switch those.

Reply:
Just tried it and my files are actually wanting .jsx for both of these, maybe because I renamed them from .js to .jsx yesterday with that error